cosmos-gui/public/stylesheets/style.styl

```diff
@@ -70,6 +70,12 @@ header
     padding 0px 10px
     margin-bottom 10px
 
+  a
```

Comment:
Again, when learning how to define styling in Node.js I found a lot of blogs and Internet places talking about Stylus and suggesting it was the better option for this purpose... So I followed them :) I found it easy to use, btw.